src/locales/es.js

```diff
@@ -27,6 +27,8 @@ export default {
   'app.legend': 'Encuentra gente cerca y conecta',
   'app.info': `Crea un objetivo en cualquier parte del mapa, indica tu interés: Travel, Dating, Music, etc
   y empieza a conectar con personas que lo comparten.`,
+  'app.welcome.item1': `Crea un objetivo haciendo click en cualquier parte del mapa, indica el área y una temática: Travel, Dating, Music, etc.`,
```

Comment:
Can we use single quotes here? I'd use that only when using interpolation

```suggestion
  'app.welcome.item1': 'Crea un objetivo haciendo click en cualquier parte del mapa, indica el área y una temática: Travel, Dating, Music, etc.',
```